protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "Manual_ControlProto";
+
+service ManualControlService
+{
+
+    
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc SetPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
```

Comment:
I think we should add either:
- something like `action->SetFlightMode(FlightMode::PositionControl)`
- or call it `StartPositionControl` becaue it' not a "setter" as it's not setting anything.

And the comment should only describe the API and not the internal implementation, so you can keep it simple.

Reply:
If MAVLink does not specify which mode goes with the `MANUAL_CONTROL` message (and hence the user has to know it), then I think I would vote for `action->SetFlightMode`... and that would centralize it for when MAVLink flight modes get better specified. Does that make sense?

Reply:
Either remove it and then we need to add a flight mode setter to `Action`, or rename it to `StartPositionControl` as suggested above.